docs/source/concepts/provenance.rst

```diff
@@ -320,64 +319,71 @@ Let us first focus on the data provenance only (i.e., only ``input_calc`` and ``
+The default behavior then was designed to reproduce the most intiuitive outcomes for this type of usage.
+
+This behavior is basically the result of the settings of ``call_calc_forward=True`` and ``call_work_forward=True``, which makes that the inclusion of a process node will also imply the inclusion of any child or parent process node as well.
+Following this rules in a recursive way leads to the command affecting all the processes within any given workflow: in this way, nodes that are sub-processes of a given highest-level workflow will end up grouped together, in the sense that (by default) they will all be affected in the same way when deleting or exporting.
 
-Enabling the optional rules in these cases overrides this default behavior.
-To better illustrate this, we consider the following graph and we focus on the deletion feature only (similar considerations apply when exporting):
+More freedom to further customize the selection of sections to export or delete is available through the specific toggable flags for each functionality (although the final sections must always comply with the non-toggable rules, see above).
+However, this usually requires a deeper understanding of the traversal rules and may imply a more thorough analysis of the particular graph.
+To better illustrate this, we will now consider the aplication of the deletion procedure to the following graph:
 
 .. _delexp_example02:
 .. image:: include/images/delexp_example02.png
    :scale: 80%
 
-As you can see, |W_1| and |W_2| describe two similar but independent procedures (e.g., two tests run for the same research project), but launched by a single parent workflow |W_0|. It might be the case, therefore, that one would like to delete information from one of them without affecting the other (e.g., if one of the tests was later deemed unnecessary).
-In this particular case, just targeting |C_1| with the default behavior gives the following result, that is probably the desired final state of the database (in the following figures, the dash-circled node is the targeted one, and nodes highlighted in red are those that are eventually deleted):
+As you can see, |W_1| and |W_2| describe two similar but independent procedures that were launched by a single parent workflow |W_0|.
+A typical user would have obtained this by directly running this workflow |W_0| to obtain the results |D_3| and |D_4| from the inputs |D_1| and |D_2|, and may even be unaware of the internal division of |W_0| into two sub-Workflows |W_1| and |W_2|.
+Hence, if the user considers the workflow (meaning, the whole set of nodes produced by it) no longer necessary, the intuitive thing to do in order to remove it from its database would be by targeting the workflow node |W_0| for deletion.
+Indeed, this would produce the desired result:
 
-.. _delexp_example02a:
-.. image:: include/images/delexp_example02-a00.png
+.. _delexp_example02-a01:
+.. image:: include/images/delexp_example02-a01.png
```

Comment:
Is there a need for this figure, where both D3 and D4 are targeted?

To me it would seem to suffice (and make things shorter) to keep only the next figure, which is identical except that only D3 is targeted.